src/zinc/os/debug.rs

```diff
@@ -77,7 +77,7 @@ mod internal {
   /// A token to ensure the life of the reference to the debugging output backend
   /// doesn't outlive the backend itself.
   #[must_use]
-  pub struct Token<'a> {
+  pub struct Token {
```

Comment:
Not certain about this removal, was not used, so there was an error . The another option is to use phantom data feature?